student-source/students/accessing-codio/paying.rst

```diff
@@ -62,13 +62,37 @@ If you have any problems contact us through your Codio account or email us at he
 Problems setting up a new subscription plan
 -------------------------------------------
 
-If you are seeing the message 'Do not Honor', then it is likely that your bank is unwilling to accept the transaction but they are not giving the specific reason to our processing company. 
+Secure online credit card transactions are processed in real-time on behalf of Codio by the authorized merchant payment service provider Braintree using the highest levels of security available on the market.
 
-You should contact your card issuing bank for more details of the exact reason. 
+Sometimes online credit card transactions would fail - the reasons for such failure may simply be the use of a wrong card number or the card limit has been reached, ad-blockers interferring, 3rd Party Cookies not allowed etc. In other cases the reasons are less obvious and may relate to the ever increasing fraud prevention and/or other security measures implemented by credit card companies to protect the cardholder against card abuse.
 
-We have heard of a few cases, especially from US banks where they do not allow international transactions by default but when you are able to explain to them what you are looking to do, they usually are able to allow it and you can then subscribe with that card but if they won't, try a different card.
+It should be understood that Codio has no control over such transaction refusal. Your credit card company would even not disclose to Codio the exact reason for your transaction to fail.
 
-If you are seeing General error in Cardinal, this error can be caused due to adblockers or cookies being blocked
+This means that unfortunately Codio cannot assist in solving the issue which is strictly a matter between the card issuing credit card company (who refused to authorize the transaction) and the credit card holder (who initiated the process to have the amount of the transaction charged to his/her credit card).
+
+In case of a transaction failure, Codio recommends to first try using an alternative credit card, or to give it another try with the same credit card again but at a later time. In most cases this would solve the issue.
+
+Do carefully check the card details before confirming the transaction for processing. Also ensure your card is actually a credit card and not just a debit card. 
```

Comment:
Carefully check the card details before confirming the transaction for processing. Also ensure your card is actually a credit card and not just a debit card.